packages/google_maps_flutter/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -8,7 +8,6 @@
 
```

Comment:
don't checkin xcode file changes.

Reply:
@awazgyawali Don't think I have permission to mark these as resolved. I suggest you take a look at my commit and resolve the issue if you agree with the changes.

Reply:
See commit a1982e4. I believe it should resolve this issue.